crates/pop-parachains/src/build.rs

```diff
@@ -1,13 +1,418 @@
 // SPDX-License-Identifier: GPL-3.0
 
+use crate::Error;
+use anyhow::Result;
 use duct::cmd;
-use std::path::PathBuf;
+use pop_common::replace_in_file;
+use serde_json::Value;
+use std::{
+	collections::HashMap,
+	fs,
+	path::{Path, PathBuf},
+};
+use toml_edit::DocumentMut;
 
-/// Build the parachain located in the specified `path`.
-pub fn build_parachain(path: &Option<PathBuf>) -> anyhow::Result<()> {
+/// Build the parachain.
+///
+/// # Arguments
+/// * `path` - Location of the parachain project.
+pub fn build_parachain(path: &Option<PathBuf>) -> Result<(), Error> {
 	cmd("cargo", vec!["build", "--release"])
 		.dir(path.clone().unwrap_or("./".into()))
 		.run()?;
+	Ok(())
+}
+
+/// Get the path to the node release binary based on the project path.
+///
+/// # Arguments
+/// * `path` - Location of the parachain project.
+pub fn binary_path(path: Option<&Path>) -> Result<PathBuf, Error> {
+	let node_name = parse_node_name(path)?;
+	let release_path = path.unwrap_or(Path::new("./")).join("target/release");
```

Comment:
Removed the hardcoded value from the library function. 
`pop-cli` will now indicate this path.  For now, we are only building parachains in release mode, not debug, so we can assume it will be this path, right?. Code here: https://github.com/r0gue-io/pop-cli/blob/alex/build-generate-files/crates/pop-cli/src/commands/build/parachain.rs#L47

Reply:
A library should be as flexible as possible, within constraints, so that it can easily flex with changes in requirements by library users to remain useful. Adding a release parameter is super simple and makes the code way more flexible. Strictly speaking, we should probably have profiles rather than just debug/releease, but can add that later.

Reply:
Makes sense to have `Profiles` from now, so won't be a breaking change.